packages/nimble-components/src/anchor-tree-item/index.ts

```diff
@@ -0,0 +1,171 @@
+import { attr } from '@microsoft/fast-element';
+import {
+    DesignSystem,
+    AnchorOptions,
+    StartEnd,
+    applyMixins,
+    isTreeItemElement,
+    TreeItem as FoundationTreeItem
+} from '@microsoft/fast-foundation';
+import { keyArrowLeft, keyEnter } from '@microsoft/fast-web-utilities';
+import { AnchorBase } from '../anchor-base';
+import { styles } from './styles';
+import { template } from './template';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-anchor-tree-item': AnchorTreeItem;
+    }
+}
+
+/**
+ * A nimble-styled anchor tree item
+ */
+export class AnchorTreeItem extends AnchorBase {
+    /**
+     * When true, the control will appear selected by user interaction.
+     * @public
+     * @remarks
+     * HTML Attribute: selected
+     */
+    @attr({ mode: 'boolean' })
+    public selected = false;
+
+    /**
+     * When true, the control will be immutable by user interaction. See {@link https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled | disabled HTML attribute} for more information.
+     * @public
+     * @remarks
+     * HTML Attribute: disabled
+     */
+    @attr({ mode: 'boolean' })
+    public disabled = false;
+
+    /**
+     * Whether the item is focusable
+     *
+     * @internal
+     */
+    public focusable?: boolean;
+
+    /**
+     * Indicates if the tree item is nested
+     *
+     * @internal
+     */
+    public nested?: boolean;
+
+    /**
+     *  Readonly property identifying the element as a tree item
+     *
+     * @internal
+     */
+    public readonly isTreeItem: boolean = true;
```

Comment:
Is this used?

Reply:
Yes, it is read by the FAST tree view.

Reply:
Actually, it is only used in newer versions of FAST. Removing.